app/structures/structures.go

```diff
@@ -21,6 +21,9 @@ type ItemPermissions struct {
 	CanEdit string `json:"can_edit"`
 	// required: true
 	IsOwner bool `json:"is_owner"`
+	// appears only in services that support this permission, and only if it is set
+	// required: true
```

Comment:
if it may not appear, it is not "required"

Reply:
fixed